catalog/mgmp/catalog-transformer-mgmp/src/main/java/org/codice/alliance/catalog/mgmp/MgmpTransformer.java

```diff
@@ -0,0 +1,678 @@
+      String[] nations =
+          StringUtils.substringAfter(caveatString, MgmpConstants.RELEASABLE_TO + " ")
+              .split(COUNTRY_SEPARATOR);
+      List<String> nationsList = Arrays.asList(nations);
+      metacard.setAttribute(releasability, (Serializable) nationsList);
+      metacard.setAttribute(disseminationControls, MgmpConstants.RELEASABLE_TO);
+    } else if (caveatString.endsWith(MgmpConstants.EYES_ONLY)) {
+      String[] nations =
+          StringUtils.substringBefore(caveatString, " " + MgmpConstants.EYES_ONLY)
+              .split(COUNTRY_SEPARATOR);
+      List<String> nationsList = Arrays.asList(nations);
+      metacard.setAttribute(releasability, (Serializable) nationsList);
+      metacard.setAttribute(disseminationControls, MgmpConstants.EYES_ONLY);
+    } else if (caveatString.endsWith(MgmpConstants.EYES_DISCRETION)) {
+      String[] nations =
+          StringUtils.substringBefore(caveatString, " " + MgmpConstants.EYES_DISCRETION)
+              .split(COUNTRY_SEPARATOR);
+      List<String> nationsList = Arrays.asList(nations);
+      metacard.setAttribute(releasability, (Serializable) nationsList);
+      metacard.setAttribute(disseminationControls, MgmpConstants.EYES_DISCRETION);
+    }
+  }
+
+  protected void addMetacardResourceSecurityClassification(
+      final XstreamPathValueTracker pathValueTracker, MetacardImpl metacard) {
+
+    Map<String, String> securityMappingList = mgmpConverter.getSecurityMap();
+
+    String firstValue =
+        pathValueTracker.getFirstValue(toPath(MgmpConstants.RESOURCE_SECURITY_PATH));
+    if (StringUtils.isNotEmpty(firstValue)) {
+      if (securityMappingList == null || securityMappingList.get(firstValue) == null) {
+        metacard.setAttribute(Security.RESOURCE_CLASSIFICATION, firstValue);
+      } else {
+        metacard.setAttribute(
+            Security.RESOURCE_CLASSIFICATION, securityMappingList.get(firstValue));
+      }
+    }
+  }
+
+  protected void addMetacardMetadataSecurityClassification(
+      final XstreamPathValueTracker pathValueTracker, MetacardImpl metacard) {
+    Map<String, String> securityMappingList = mgmpConverter.getSecurityMap();
+
+    String firstValue =
+        pathValueTracker.getFirstValue(toPath(MgmpConstants.METADATA_SECURITY_PATH));
+    if (StringUtils.isNotEmpty(firstValue)) {
+      if (securityMappingList == null || securityMappingList.get(firstValue) == null) {
+        metacard.setAttribute(Security.METADATA_CLASSIFICATION, firstValue);
+      } else {
+        metacard.setAttribute(
+            Security.METADATA_CLASSIFICATION, securityMappingList.get(firstValue));
+      }
+    }
+  }
+
+  protected void addMetacardDataQuality(MetacardImpl metacard) {
+
+    try (InputStream inputStream = getSourceInputStream()) {
+      DocumentBuilderFactory domFactory = DocumentBuilderFactory.newInstance();
+      domFactory.setNamespaceAware(false);
+      try {
+        domFactory.setFeature(
```

Comment:
There are some other settings that have been adopted for security reasons: https://github.com/codice/ddf/blob/master/platform/util/platform-util/src/main/java/org/codice/ddf/platform/util/XMLUtils.java#L358